connectors/src/lib/temporal_monitoring.ts

```diff
@@ -254,3 +255,33 @@ export class ActivityInboundLogInterceptor
     }
   }
 }
+
+export class ActivityOutboundLogInterceptor
+  implements ActivityOutboundCallsInterceptor
+{
+  private readonly provider: ConnectorProvider;
+
+  constructor(provider: ConnectorProvider) {
+    this.provider = provider;
+  }
+
+  getLogAttributes(
+    input: GetLogAttributesInput,
+    next: Next<ActivityOutboundCallsInterceptor, "getLogAttributes">
+  ): Record<string, unknown> {
+    return next({
+      ...input,
+      provider: this.provider,
+    });
+  }
+
+  getMetricTags(
+    input: MetricTags,
+    next: Next<ActivityOutboundCallsInterceptor, "getMetricTags">
+  ): MetricTags {
+    return next({
+      ...input,
+      provider: this.provider,
+    });
+  }
```

Comment:
nit: comment as it's surprising to see `getMetricTags` in `ActivityOutboundLogInterceptor`